components/LiveUpdate.js

```diff
@@ -65,18 +65,19 @@ const LiveUpdate = ({ liveUpdates, loading }) => {
 
   return (
     <>
-      <Title> Live Updates <DurationText> {durationText} </DurationText></Title>
+      <Title> Weekly Updates <DurationText> {durationText} </DurationText></Title>
       {loading && (
         <>
           <div style={{ marginTop: '1rem', marginBottom: '1rem' }}>
-            <Skeleton count={5}/>
+            <Skeleton count={1}/>
           </div>
-          <Skeleton count={5}/>
+          <Skeleton count={1}/>
         </>
       )}
-      {liveUpdates && liveUpdates.map(({ title, content, timestamp }) => (
+      <ContentIndent>
+      {liveUpdates && [liveUpdates[0]].map(({ title, content, timestamp }) => (
```

Comment:
since we are only using the first element in `liveUpdates`, there's no need to wrap this in an array and use `.map`.
u sld only render only 1 `UpdateWrapper`
also, since this block of code is wrapped inside `ContentIndent`, u sld indent this entire block